cleanlab/datalab/issue_manager/noniid.py

```diff
@@ -164,6 +164,16 @@ def find_issues(self, features: Optional[npt.NDArray] = None, **kwargs) -> None:
             },
         )
 
+        num_issues = self.issues[f"is_{self.issue_name}_issue"].sum()
+        if self.p_value >= 0.05:
+            # Set all issues to False if the p-value is greater than 0.05
+            self.issues[f"is_{self.issue_name}_issue"] = False
+        elif num_issues == 0:
+            # Make the lowest scoring example an issue if there are no issues
+            self.issues.loc[
+                self.issues[self.issue_score_key].idxmin(), f"is_{self.issue_name}_issue"
+            ] = True
+
```

Comment:
why not get rid of these lines?
https://github.com/cleanlab/cleanlab/blob/e0a34113942fcba3986a0adebb85da0097709fe6/cleanlab/datalab/issue_manager/noniid.py#L160

And only define self.issues here in the first place?

Reply:
Right. I've reduced the heuristic to only update an `issue_mask`, then `self.issues` is defined only once here.

Thanks!